src/platform-integration/ios/app-extensions.md

```diff
@@ -119,30 +141,32 @@ such as sharing a picture to create a new post on a social media app.
     </div>
 </figure>
 
-Starting from version 3.16, Flutter supports
-building Flutter UI for app extensions.
-To create the UI for
-an app extension using Flutter, you must use an extension-safe
-`Flutter.xcframework` and embed the `FlutterViewController`
-as described in the following section.
+As of the 3.16 release, you can build
+Flutter UI for an app extension,
+though you must use an extension-safe
+`Flutter.xcframework` and embed the
+`FlutterViewController` as described in
+the following section.
 
 {{site.alert.note}}
   Due to the memory limitations of app extensions,
-  it is only recommended to use Flutter to build app extension UI
+  it's only recommended to use Flutter to build app extension UI
   for extension types that have memory limits larger than 100MB.
-  For example, share extensions which have a 120MB memory limit.
+  For example, share extensions has a 120MB memory limit.
 
   In addition, Flutter uses extra memory in debug mode.
-  Therefore, Flutter does not fully support running app extensions in
-  debug mode on physical devices when used to build extension UI.
-  As an alternative, use an iOS simulator to test your extension in debug mode.
+  Therefore, Flutter doesn't fully support running
+  app extensions in debug mode on physical devices
+  when used to build extension UI. As an alternative,
+  use an iOS simulator to test your extension in debug mode.
```

Comment:
**issue:** This needs to be stated more clearly. Does Flutter run app extensions in debug mode or not? If so, write the difficulty that might be experienced when doing so. If it doesn't, say it doesn't run it.

Consider something more direct:

```suggestion
  Therefore, when used to build the iOS extension UI in debug mode,
  Flutter doesn't run iOS app extensions on physical devices.
  As an alternative, use an iOS simulator to test your extension in debug mode.
```

Reply:
It depends on the size of the app.